bases/br_bd_diretorios_brasil/etnia_indigena/table_config.yaml

```diff
@@ -90,23 +82,17 @@ auxiliary_files_url:
 # Url da tabela de arquitetura no GCP Storage.
 architecture_url:
 
-# A tabela tem colunas que precisam de dicionário?
-# Opções: yes, no.
-covered_by_dictionary: no
+source_bucket_name: bd-projetos
```

Comment:
Se os dados ainda não estiverem no BQ e vc for subir com o nosso pacote, esse campo e os próximos dois tem que ser `basedosdados-dev`